liferay-data-engine/liferay-portal
# LPS-121872 Migrate v2 usages in dynamic-data-lists/dynamic-data-lists-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

Creating, managing and deleting dynamic data lists should work as
expected.

**Test plan**:
- Navigate to **Content & Data** > **Dynamic Data Lists**
- From the management toolbar, make sure you can create a new Dynamic
Data List
- From the management toolbar, make sure you can delete one or more Dynamic
Data List(s)
- Click in a Dynamic Data List and make sure that you can add new items
- Click in one of the Dynamic Data List's items and make sure you can
edit it
